apps/hash-search-loader/package.json

```diff
@@ -13,7 +13,7 @@
     "start": "cross-env NODE_ENV=production node --max-old-space-size=2048 ./node_modules/.bin/ts-node --transpile-only ./src/index.ts"
   },
   "dependencies": {
-    "@hashintel/hash-api": "0.0.0-private",
+    "@apps/hash-api": "0.0.0-private",
```

Comment:
These imports are wrong because we can only import from libs. Fixing is out of scope though.

Reply:
[Asana task](https://app.asana.com/0/1203193968491715/1203774620719383/f) (internal)